subsys/bluetooth/host/audio/chan.c

```diff
@@ -1219,6 +1219,31 @@ int bt_audio_unicast_group_create(struct bt_audio_chan *chans,
 	return 0;
 }
 
+int bt_audio_unicast_group_delete(struct bt_audio_unicast_group *unicast_group)
+{
+	struct bt_audio_chan *chan;
+
+	CHECKIF(unicast_group == NULL) {
+		BT_DBG("unicast_group is NULL");
+		return -EINVAL;
+	}
+
+	SYS_SLIST_FOR_EACH_CONTAINER(&unicast_group->chans, chan, node) {
+		if (chan->state != BT_AUDIO_CHAN_IDLE &&
+		    chan->state != BT_AUDIO_CHAN_CONFIGURED) {
+			BT_DBG("chan %p is not idle", chan);
```

Comment:
Proposal: "[...] state is neither idle nor configured"

Reply:
Made it a bit more generic and added logging of the state that is invalid.